### ISSUE TITLE
Calendar: Name is not defined for the dialog which appears on pressing enter on click for calendar button.

### ISSUE THREAD

**Issue by dzearing** [OfficeDev/office-ui-fabric-react#11144]
Tracking all of the issues brought up with the 03/15 accessibility pass on Calendar fabric control
Details on individual bugs can be found on [Sharepoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B8cfe67c5-ce76-4083-b60e-4190dbd38206%7D&action=default&uid=%7B8CFE67C5-CE76-4083-B60E-4190DBD38206%7D&ListItemId=1872&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble)

**Comment by lorejoh12** [comment]
I'm unsure about what the correct fix is here. What name should we be using? I'm guessing by "name", this bug is about there being no aria-label on the dialog that pops up in the examples. Technically that's a bug with the example itself, not the Calendar component, because the example is what wraps the calendar in the Popup, but if this is something that needs to be fixed, it'll need to be fixed in the DatePicker component, so we should get some clarity on expected behavior here.

**Comment by lorejoh12** [comment]
For some reason the PR merging didn't auto close the issues... closing, this should be fined now.
